README.rst

```diff
@@ -74,11 +74,19 @@ Install from source
    git clone git@github.com/eitrtechnologies/idemenv.git
    cd idemenv
 
-   # Setup venv
+   # Setup venv (linux & macos)
    python3 -m venv .venv --prompt idemenv
    source .venv/bin/activate
    pip install -e .
 
+   # Setup venv (windows)
+   python3 -m venv .venv --prompt idemenv
+   . .venv\Scripts\activate
+   pip install -e .
+   
+   # Note: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": 
+   # https://visualstudio.microsoft.com/visual-cpp-build-tools/
+
```

Comment:
This section has been a little problematic with rstcheck. How about something like the following?

```suggestion

**Linux & MacOS:**
 
.. code-block:: bash

   # Setup venv
   python3 -m venv .venv --prompt idemenv
   source .venv/bin/activate
   pip install -e .

**Windows:**

.. code-block::

   # Setup venv
   python3 -m venv .venv --prompt idemenv
   . .venv\Scripts\activate
   pip install -e .

.. note::
   Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": 
   https://visualstudio.microsoft.com/visual-cpp-build-tools/

```